src/components/link/daikin-link.ts

```diff
@@ -0,0 +1,151 @@
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import tailwindStyles from "../../tailwind.css?inline";
+
+const cvaLink = cva(
+  [
+    "flex",
+    "justify-center",
+    "items-center",
+    "gap-0.5",
+    "w-fit",
+    "h-fit",
+    "font-daikinSerif",
+    "rounded-[1px]",
+    "relative",
+
+    "focus-visible:outline-none",
+
+    "focus-visible:before:block",
+    "focus-visible:before:w-[calc(100%+2px)]",
+    "focus-visible:before:h-[calc(100%+2px)]",
```

Comment:
Maybe we can use inset? (not tested)

```suggestion
    "focus-visible:before:inset-[-1px]",
```